tests/webtests/EntityCopyTest.php

```diff
@@ -153,13 +153,29 @@ public function test_copy_to_production_does_not_create_new_entity()
         );
         $this->testMockHandler->append(new Response(200, [], json_decode($response)));
 
-        $this->client->request('GET', "/entity/copy/d645ddf7-1246-4224-8e14-0d5c494fd9ad/production");
+        $this->client->request('GET', "/entity/copy/1/d645ddf7-1246-4224-8e14-0d5c494fd9ad/production");
```

Comment:
I think it would be better to fix the test completely because now the `assertEmpty` is returning true on an empty array and results in a false positive.

Reply:
Yes, I agree

Reply:
:+1: thanks!